.github/workflows/rosa-cluster-create.yml

```diff
@@ -23,6 +23,15 @@ on:
       region:
         description: 'The AWS region to create the cluster in. Defaults to "vars.AWS_DEFAULT_REGION" if omitted.'
         type: string
+      infinispan_operator_olm_catalog_source:
+        type: string
+        default: community-operators
+      infinispan_operator_olm_name:
+        type: string
+        default: infinispan
+      infinispan_operator_olm_channel:
+        type: string
+        default: stable
```

Comment:
Inifinspan installation happens within the rosa-cluster-create workflow so passing the custom params here.